tests/functional/timings.test.js

```diff
@@ -391,6 +392,76 @@ function runClsTests(loader) {
     }
   })
 
+  testDriver.test(`${loader} agent collects cls attribute when cls is 0`, supportedCls, function (t, browser, router) {
+    t.plan(2)
+
+    // load page without any expected layout shifts
+    let url = router.assetURL('instrumented.html', { loader: loader })
+    let loadPromise = browser.safeGet(url).catch(fail)
+
+    Promise.all([loadPromise, router.expectRum()])
+      .then(() => {
+        let timingsPromise = router.expectTimings()
+        let domPromise = browser.get(router.assetURL('/'))
+        return Promise.all([timingsPromise, domPromise]).then(([data, clicked]) => {
+          return data
+        })
+      })
+      .then(({body, query}) => {
+        const timings = querypack.decode(body && body.length ? body : query.e)
+
+        var unload = timings.find(t => t.name === 'unload')
+        var cls = unload.attributes.find(a => a.key === 'cls')
+
+        t.ok(unload, 'there should be an unload timing')
+        t.equal(cls.value, 0, 'cls value should be a perfect score of 0')
+
+        t.end()
+      })
+      .catch(fail)
+
+    function fail (err) {
+      t.error(err)
+      t.end()
+    }
+  })
+
+  testDriver.test(`${loader} agent does not collect cls attribute on unsupported browser`, unsupportedCls.and(reliableFinalHarvest), function (t, browser, router) {
```

Comment:
I think this test should match only browsers that do not support CLS (supportedCls.inverse()).

Reply:
Never mind, you got it :)